src/styles/atoms/_atoms.button.scss

```diff
@@ -280,6 +280,7 @@
 
   &.a-button--large {
     font-size: rem($btn-font-size-lg);
+    line-height: $spacer-xl;
```

Comment:
Voor `.a-button`, `.a-button--small` en `.a-button--tiny` was er specifiek een line-height gedefinieerd. Voor `.a-button--large` was dit nog niet het geval. Daarom stond de label bij onderstaande knop niet mooi in het midden.

<a href="#" class="a-button a-button--large has-icon-left">
    <i class="fa fa-arrow-left"></i>Dikke knop
</a>